### PR TITLE
Add small improvement on throttle function

### PR DESCRIPTION
**Pull request by rleite**
Most important part is the instantiation of a new Date object instead of just using the Date.now() method.

### CODE REVIEW

File: src/infinite-scroll.js [hunk 1]
```diff
@@ -108,19 +108,23 @@ angular.module(MODULE_NAME, [])
...
-          previous = new Date().getTime();
-          $interval.cancel(timeout);
+        function clear() {
+          if (timeout) {
+            $interval.cancel(timeout);
+          }
           timeout = null;
+        }
+        function later() {
+          clear();
```
Comment (graingert) [inline src/infinite-scroll.js]: why do you need to call clear in this "later" function?

Comment (graingert) [inline src/infinite-scroll.js]: Please include a test that fails without it.

Comment (rleite) [inline src/infinite-scroll.js]: 1. it was being "cleared" before but your right doesn't make sense.
2. This is just a small teak nothing changes in terms of ui to test.

**Comment by rleite** [comment]
Updated

**Pull request closed without merging**